api/envoy/extensions/transport_sockets/tls/v3/tls.proto

```diff
@@ -124,8 +124,20 @@ message DownstreamTlsContext {
   OcspStaplePolicy ocsp_staple_policy = 8 [(validate.rules).enum = {defined_only: true}];
 }
 
+// TLS key log configuration.
+message TlsKeyLog {
```

Comment:
This would benefit from a `RuntimeFeatureFlag` to enable/disable without reconfiguring.

Reply:
I will add

Reply:
will add